src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension/package.json

```diff
@@ -0,0 +1,52 @@
+{
+	"name": "ms-blazorwasm-companion",
```

Comment:
ctrl-k-d?

Reply:
Hm. This is already formatted. Is there something in particularly that is off? Might be my VSCode config...

Reply:
Oop. Nevermind. This did actually need formating.